src/main/domain/vendorlist/VendorListHelper.js

```diff
@@ -0,0 +1,48 @@
+export class VendorListHelper {
+  haveAllValuesTo({object, valueToVerify}) {
+    if (Object.entries(object).length === 0) {
+      return valueToVerify === false
+    }
+    // eslint-disable-next-line no-unused-vars
+    for (const [_, value] of Object.entries(object)) {
+      if (valueToVerify !== value) {
+        return false
+      }
+    }
+    return true
+  }
+
+  setAllVendorsTo({vendorList, valueToSet}) {
+    const newVendor = {
+      consents: {},
+      legitimateInterests: {}
+    }
+    Object.entries(vendorList).forEach(([key, value]) => {
```

Comment:
If you are not checking the value of vendorList object here you can use Object.keys

Reply:
done. Thanks!